src/environment-data-provider.ts

```diff
@@ -12,3 +12,11 @@ export function getEnvironmentData(
         targetPageTitle: axeResults.targetPageTitle,
     };
 }
+
+export function getEnvironmentDataFromEnvironment(): EnvironmentData {
+    return {
+        timestamp: new Date().toISOString(),
+        targetPageUrl: window.location.href,
+        targetPageTitle: '',
```

Comment:
We can probably fill this in at this point from `window.document.title` or similar.